.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,11 @@
+# Issue: Feature
+> Note: The quickest way to see a new feature would be to submit a pull request, which will be reviewed when possible.
```

Comment:
strictly speaking this is not absolutely true :) we have an architecture of candid in mind and outside users might not understand this fully. So a PR might lead to lengthy discussions an change requests. But i agree, we should not discourage anybody from proposing a PR. 

Instead, perhaps we should say something along the lines: "if you believe this feature requires a small change, feel free to propose a PR, which will be reviewed as soon as possible"

Reply:
Ah ok, I'll update to your recommendation.